### PR TITLE
Run bach Athena tests in CI

### PR DESCRIPTION
**Pull request by thijs-obj**
Run Bach Athena tests (1 so far) in CI.

Also added the secrets as repository level secrets in github. I used the account that is in 1password, which I created according to the doc here: https://github.com/objectiv/objectiv-analytics/pull/1097/files#diff-d396beb544e288b0ba60a663e94d78cc8f10496804421e2c9881d0acd8fbe98a

Please have a look @borft @KathiaBarahona

### CODE REVIEW

**Review by borft — APPROVED** [review]
lgtm

File: .github/workflows/bach-tests.yml [hunk 1]
```diff
@@ -90,11 +92,25 @@ jobs:
       - name: Install bigquery dependencies
         run: |
           pip install -e bach[bigquery]
-      - name: Unit tests big-query
+      - name: Unit tests BigQuery
```
Comment (KathiaBarahona) [inline .github/workflows/bach-tests.yml:95]: or capitalize `postgres` in lines 80 and 84 :P just for following the same pattern
```suggestion
      - name: Unit tests bigquery
```

File: .github/workflows/bach-tests.yml [hunk 2]
```diff
@@ -90,11 +92,25 @@ jobs:
...
+        run: |
+          pip install -e bach[athena]
+      - name: Unit tests Athena
+        run: |
+          cd bach
+          pytest --athena tests/unit
+      - name: Functional tests Athena
+        run: |
+          cd bach
+          pytest -n 4 --dist loadgroup --athena tests/functional
```
Comment (KathiaBarahona) [inline .github/workflows/bach-tests.yml:116]: maybe we can run without parallelization and when we progress with the tests support we change it. So, probably an issue for now is enough

**Review by KathiaBarahona — APPROVED** [review]
lgtm overall, just couple of comments

File: .github/workflows/bach-tests.yml (hunk 1 above)
Comment (thijs-obj) [reply .github/workflows/bach-tests.yml:95]: 04c812b3496f9484445e9f6fca94a167ae0f15f3

File: .github/workflows/bach-tests.yml (hunk 2 above)
Comment (thijs-obj) [reply .github/workflows/bach-tests.yml:116]: One worker would be OK for now, but we should get benefit from multiple workers quite soon.

**Question:** What do you mean by "an issue for now is enough"?

**Pull request merged**